test/helpers/ssh_command.go

```diff
@@ -248,30 +248,62 @@ func (client *SSHClient) RunCommandContext(ctx context.Context, cmd *SSHCommand)
 		panic("nil context provided to RunCommandContext()")
 	}
 
-	session, err := client.newSession()
-	if err != nil {
-		return err
-	}
+	var (
+		session    *ssh.Session
+		err        error
+		sessionErr error
+	)
+
+	sessionErrChan := make(chan error, 1)
+
+	go func(errChan chan error) {
```

Comment:
It always confuses me why you pass some parameters to the go routine but not all.

Reply:
It is confusing, and I should be better at being consistent there.

I wanted to make the goroutine a closure on `session`. I could alternatively pass a pointer to a pointer, or remove the error channel parameter for consistency. I'll do the latter.

Reply:
Removed parameter for consistency.